src/Controls/tests/DeviceTests/Elements/Editor/EditorTests.Android.cs

```diff
@@ -57,5 +59,39 @@ public async Task CursorPositionPreservedWhenTextTransformPresent()
 
 			Assert.Equal(2, editor.CursorPosition);
 		}
+
+		// This test will only run if the Android Manifest of the Controls.DeviceTests project is edited to have android:supportsRtl="false"
+		[Fact(DisplayName = "Horizontal text aligned when RTL is not supported")]
+		public async Task HorizontalTextAlignedWhenRtlIsFalse()
```

Comment:
Should the Editor, Entry and Search tests be added to the TextInputHandlerTests base test?

src/Core/tests/DeviceTests.Shared/HandlerTests/TextInput/TextInputHandlerTests.cs

Then we only have to duplicate for Label. And if/when we add a new input type, we get magic coverage.

Reply:
Done!